### PR TITLE
Add os.chmod

### PR DESCRIPTION
**Pull request by suewoon**
Added `os.chmod`, which works on unix platforms only.

### CODE REVIEW

File: vm/src/stdlib/os.rs [hunk 1]
```diff
@@ -990,6 +1006,7 @@ pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
         "lstat" => ctx.new_rustfunc(os_lstat),
         "getcwd" => ctx.new_rustfunc(os_getcwd),
         "chdir" => ctx.new_rustfunc(os_chdir),
+        "chmod" => ctx.new_rustfunc(os_chmod),
```
Comment (youknowone) [inline vm/src/stdlib/os.rs]: Because `os_chmod` is only defined in unix, this line causes error in windows.

**Comment by youknowone** [comment]
About breaking in travis, `cargo fmt --all` will fix formatting.
Check Details about clippy warning.

File: vm/src/stdlib/os.rs [hunk 2]
```diff
@@ -950,6 +965,7 @@ pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
         SupportFunc::new(vm, "chdir", os_chdir, Some(false), None, None),
         // chflags Some, None Some
         // chmod Some Some Some
```
Comment (youknowone) [inline vm/src/stdlib/os.rs]: This line is now removable

File: vm/src/stdlib/os.rs [hunk 3]
```diff
@@ -862,6 +863,26 @@ fn os_chdir(path: PyStringRef, vm: &VirtualMachine) -> PyResult<()> {
...
 }
 
+#[cfg(unix)]
+fn os_chmod(
+    path: PyStringRef,
+    dir_fd: DirFd,
+    mode: u32,
+    follow_symlinks: FollowSymlinks,
+    vm: &VirtualMachine,
+) -> PyResult<()> {
```
Comment (youknowone) [inline vm/src/stdlib/os.rs]: maybe importing here will work

**Comment by palaviv** [comment]
@coolreader18 was this a mistake? This PR does not pass the CI.

**Comment by coolreader18** [comment]
It showed that it was passing for me, the mobile app I use is somewhat buggy.

**Comment by coolreader18** [comment]
Or actually, it only showed Travis, I didn't see the Azure status. I'll commit a quick fix to master.

**Pull request merged**